### ISSUE TITLE
The plotly_direct command doesn't provide session state to the dash callback using Dash 2.0

### ISSUE THREAD

**Issue by EmperorPenguin117** [GibbsConsulting/django-plotly-dash#374]
This HTML code
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
    {%load plotly_dash%}
    {% plotly_header %}
</head>

<body>
{% plotly_direct name="SimpleExample"%}

</body>
{% plotly_footer %}
</html>
```

with this in app views.py
```
def detail(request, pid):
    flight = get_object_or_404(Flight, pk=pid)
    session = request.session
    django_plotly_dash = session.get('django_plotly_dash', {})
    django_plotly_dash['pid'] = pid
    session['django_plotly_dash'] = django_plotly_dash

    context = {'flight': flight, 'pid': pid}
    return render(request, 'flights/detail.html', context)
```

works in Dash 1.21 but not in Dash 2.0. In 2.0 `session_state` is none in the dash callback.
```
def callback_color(selection1, session_state=None):
    if session_state is None:
        raise NotImplementedError("Cannot handle a missing session state")
    pid = session_state.get('pid', None)
    print(pid)
    return get_hr_plot(models.Flight.objects.get(id=pid))
```

**Comment by GibbsConsulting** [comment]
@EmperorPenguin117 we don't support Dash 2.0 yet - essentially because of how Dash 2.0 handles callbacks. See #354

Did the incompatible version requirements not raise any error or requirements when you installed the django-plotly-dash or dash packages?

**Comment by GibbsConsulting** [comment]
Should now be resolved with the v2.0.0 release - please reopen or create a new issue if not.
